lib/controllers/DashboardController.js

```diff
@@ -121,8 +121,10 @@ class DashboardController extends BaseController {
     const service_id = req.params.service_id;
     const plan_id = req.params.plan_id;
     logger.info(`Validating service '${service_id}' and plan '${plan_id}'`);
+    const operation = utils.decodeBase64(_.get(req, 'query.operation') || 'bnVsbA==');
```

Comment:
please move `bnVsbA==` to constants as it is difficult to understand what this string means.

Reply:
done